com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/ShaderGraphUIBlock.cs

```diff
@@ -143,7 +174,10 @@ void DrawShadowMatteToggle()
             materials[0].SetFloat(HDMaterialProperties.kShadowMatteFilter, HDShadowUtils.Asfloat(finalFlag));
         }
 
-        void DrawDiffusionProfileUI()
+        /// <summary>
+        /// Draw the built-in exposed diffusion profile when a material have sub-surface scattering or transmission.
```

Comment:
Draw the built-in exposed Diffusion Profile when a material uses sub-surface scattering or transmission.